internal/commands/secrets/create_inputs.go

```diff
@@ -0,0 +1,52 @@
+package secrets
+
+import (
+	"github.com/10gen/realm-cli/internal/app"
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/terminal"
+	"github.com/AlecAivazis/survey/v2"
+)
+
+const (
+	flagName      = "secret-name"
```

Comment:
ignore my spec now that we can't have `-` flags with more than one letter to them... the long forms of these flags should be `--name` and `--value` here (since `secret` is already known from the command itself)